app/models/authentication.rb

```diff
@@ -0,0 +1,23 @@
+class Authentication
+
+  def initialize(parser)
+    @parser = parser
+  end
+
+  def current_user(request)
+    @request = request
+
+    if !@request.headers['Authorization']
+      return nil
+    end
```

Comment:
:+1:
Just FYI: a more idiomatic ay in ruby would be 
`return nil unless @request.headers['Authorization']`

Reply:
ah yeah, I mean to go back and change it to that format and forgot